docs/DebuggingGuide.md

```diff
@@ -30,7 +30,13 @@ Additional setup guidance can be found in the [Contributing.md](https://github.c
     `git checkout -b informative-branch-name`
 
 ## Debugging
-0. Install the VS SARIF viewer for [VS 2019](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer) or [VS 2022](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer2022).
+0. Install the VS SARIF viewer for [VS 2022](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer2022).
+   
+   Select `Debug` from the `Solution Configurations` menu:
+
+   ![Solution Configurations: Debug](DebugBuild.png)
+   
+   Then run `Build Solution` from the `Build` menu.
```

Comment:
>Then run [](http://example.com/codeflow?start=3&length=8)

Not sure if this is even relevant or important at all, but when opening VS a CMake prompt pops up. I hit "do not enable", but if this is necessary to note, then a clarification here would be useful to a new person.